### PR TITLE
Arango gutierrez development

### PR DESCRIPTION
**Pull request by GodloveD**
**Description of the Pull Request (PR):**

Removing version specific info from INSTALL.md so that we don't have to try to remember to update every time we come out with a new release. 

**This is an addendum to the following GitHub pull request:**

- https://github.com/singularityware/singularity/pull/1108

**Checkoff for all PRs:**

- [x] I have read the [Guidelines for Contributing](https://github.com/singularityware/singularity/blob/master/CONTRIBUTING.md), and this PR conforms to the stated requirements.
- [ ] I have tested this PR locally with a `make test`
- [x] This PR is NOT against the project's master branch
- [x] I have added myself as a contributor to the [contributors's file](https://github.com/singularityware/singularity/blob/master/CONTRIBUTORS.md)
- [x] This PR is ready for review and/or merge

Attn: @singularityware-admin

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -10,59 +10,46 @@ $ sudo yum groupinstall "Development Tools"
...
 note: The `sudo` is very important for the `make install`. Failure to do this
 will result in a non-functioning or semi-functioning installation.
 
 ## To build an RPM of Singularity from a Git clone:
-
-```
+Here, the version of Singularity that you want to install is given in <b>&lt;version&gt;</b>.  Please substitute as necessary.  
+<pre>
+$ version=<b>&lt;version&gt;</b>
 $ git clone https://github.com/singularityware/singularity.git
```
Comment (vsoch) [inline INSTALL.md:48]: these guys render ok right?

Comment (vsoch) [reply INSTALL.md:48]: oh yeah, derp :OP 
image

**Comment by vsoch** [comment]
+1 on this. Can we remove it here as well?
https://github.com/singularityware/singularity/blob/development/.github/RELEASE_CHECKLIST.md

**Review by vsoch — APPROVED** [review]

**Comment by gmkurtzer** [comment]
Excellent, thank you!

**Pull request merged**